crates/biome_css_syntax/src/stmt_ext.rs

```diff
@@ -45,4 +45,14 @@ impl CssBlockLike {
             CssBlockLike::CssDeclarationOrRuleBlock(block) => block.items().is_empty(),
         }
     }
+
+    pub fn is_empty_without_comments(&self) -> bool {
```

Comment:
Can we add doc comments to this new function?

Reply:
Sure!